docs/deployment.md

```diff
@@ -56,6 +58,15 @@ Options:
   --timeout-keep-alive INTEGER    Close Keep-Alive connections if no new data
                                   is received within this timeout.  [default:
                                   5]
+  --keyfile TEXT                  SSL key file
+  --certfile TEXT                 SSL certificate file
+  --ssl-version INTEGER           SSL version to use (see stdlib ssl module's)
+                                  [default: 2]
+  --cert-reqs INTEGER             Whether client certificate is required (see
+                                  stdlib ssl module's)  [default: 0]
+  --ca-certs TEXT                 CA certificates file
+  --ciphers TEXT                  Ciphers to use (see stdlib ssl module's)
+                                  [default: TLSv1]
```

Comment:
I'm thinking it might make sense for us to prefix all our command line options here as "ssl-...".

Reply:
The only downside about doing this, is that it's no longer "compatible" with gunicorn.
Right now they this way `uvicorn` and `gunicorn` share the same interface. So if you want to use ssl for development, and then go to production with gunicorn, your settings don't change. I think it might make it a bit annoying for uvicorn users. What do you think?

Reply:
That's a good point.
I *think* I'm okay with that - we already differ on `--bind` vs. `--host`/`--port`.